nipype/pipeline/engine/nodes.py

```diff
@@ -1039,10 +1038,12 @@ def __init__(self, interface, iterfield, name, serial=False, nested=False, **kwa
         name : alphanumeric string
             node specific name
         serial : boolean
-            flag to enforce executing the jobs of the mapnode in a serial manner rather than parallel
+            flag to enforce executing the jobs of the mapnode in a serial
+            manner rather than parallel
         nested : boolea
-            support for nested lists, if set the input list will be flattened before running, and the
-            nested list structure of the outputs will be resored
+            support for nested lists, if set the input list will be flattened
+            before running, and the nested list structure of the outputs will
+            be resored
```

Comment:
restored

Reply:
`support for nested lists. If set, ... before running and the nested ... restored`